chore(Boilerplate): implement boilerplate for project

Angemichel12 opened the pull request:
-add controllers, database,helpers,routes,services,validations directories [Finishes #187419036]

leandreAlly commented on package.json:
```diff
@@ -0,0 +1,32 @@
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "ts-node-dev --require dotenv/config src/index.ts",
```
@Angemichel12 we need migration scripts and seeders

Angemichel12 committed:
chore(Boilerplate): implement boilerplate for project
-add controllers, database,helpers,routes,services,validations,middlewares directories
[Finishes #187419036]
